README.md

```diff
@@ -1 +1,68 @@
-# hello-world
\ No newline at end of file
+# Project Name
+
+Hello Microverse
+
+# Description the project.
+
+The Project is to get started with linter configuration.
+
+The pull request is opened to get a review on the project and check for any required changes.
+
+## Built With
+
+- HTML
+- CSS
+
+---
+
+To get a local copy up and running follow these simple example steps.
+follow>>>>>>>>>
+
+### Setup
+
+to use the project To do list please follow :
+
+> Open GitHub and go to my GitHub repository name: hello_microverse clone.
+> Click “Code” and copy the given URL.
+> click code and copy the URL.
+
+> click code and copy the URL.
+
+> Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
+
+> Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+> Syntax:
+
+$ git clone {repository URL}
+
+### Usage
+
+> open your directory math-magician/
+
+\_use vs-code or any IDE to view line code .
+
+## Authors
```

Comment:
- [ ] I think it should be better if you delete the `s` to `Authors` cause you're the owner of this project, there no other author 🙃 🙃

Reply:
Okay Thank you @cynthiainga